frontend/src/metabase/visualizations/components/skeletons/StaticSkeleton/StaticSkeleton.tsx

```diff
@@ -44,7 +52,19 @@ const StaticSkeleton = ({
         </Tooltip>
       )}
       <SkeletonTitle>{name}</SkeletonTitle>
-      {description && <SkeletonDescription>{description}</SkeletonDescription>}
+      {description && (
+        <SkeletonDescription
+          alwaysShowTooltip={hasMoreDescriptionToShow}
+          placement="bottom"
+          tooltip={
+            <Markdown disallowHeading unstyleLinks>
+              {description}
+            </Markdown>
+          }
+        >
+          {getLeadingText(defaultedDescription)}
```

Comment:
@kamilmielnik I think that we parse markdown 2 times with the current API, could that be avoided?

Reply:
e120c406717db8478c41511ec55e67781c4eadbc